shared.primitives/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,20 @@
+Manifest-Version: 1.0

```

Comment:
This file should be removed, since it is generated on every build and `META-INF` is ignored.

Reply:
Done